### ISSUE TITLE
About train on other data

### ISSUE THREAD

**Issue by ylc2580** [sfzhang15/FaceBoxes#48]
Hi,
        I want to use this method to detect people. The proportion of people is generally not 1:1. In the initial test, I did not modify the aspect_ratio in your train.prototxt. This  maybe a potential problem. 
        first,when i training about 15,000 steps, the loss did not dropped, and  it is around 6. Is this normal? Or is training steps too small? 
        Thank you very mush. -.-

**Comment by sfzhang15** [comment]
@ylc2580 
Some hyperparameters maybe need to be adjusted to detect people. The final loss is about 3 for face detection, you can try to train the original face detection to check it.
